docs/catalog.md

```diff
@@ -0,0 +1,117 @@
+| `thumbnail`     | String           | Thumbnail of the feedstock.                      |                                                                                                                                                                                                                                                                                                                                                               |
+| `tags`          | Array of Strings | Tags associated with the feedstock.              |                                                                                                                                                                                                                                                                                                                                                               |
+| `links`         | Array of Objects | Additional links related to the feedstock.       | `label`: Label of the link (Type: String)<br>`url`: URL of the link (Type: String)                                                                                                                                                                                                                                                                            |
+| `stores`        | Array of Objects | Information about where the feedstock is stored. | `id`: ID of the store (Type: String)<br>`name`: Name of the store (Type: String, optional)<br>`url`: URL of the store (Type: String)<br>`rechunking`: Rechunking information (Type: Array of Objects, optional)<br>`public`: Whether the store is public (Type: Boolean, optional)<br>`geospatial`: Whether the store is geospatial (Type: Boolean, optional) |
+
+### Example YAML Files
+
+Here's an example of a `meta.yaml` file:
+
+```yaml
+# meta.yaml
+title: "LEAP Data Library Prototype"
+description: >
+  A prototype test for the LEAP Data Library refactor
+recipes:
+  - id: "small"
+    object: "recipe:small"
+  - id: "large"
+    object: "recipe:large"
+provenance:
+  providers:
+    - name: "Julius"
+      description: "Just a guy testing some recipes. Nothing to see here."
+      roles:
+        - producer
+        - licensor
+  license: "Just a Test"
+maintainers:
+  - name: "Julius Busecke"
+    orcid: "0000-0001-8571-865X"
+    github: jbusecke
+```
+
+Here's an example of a `catalog.yaml` file:
+
+```yaml
+# catalog.yaml
+meta_yaml_url: "https://github.com/leap-stc/proto_feedstock/blob/main/feedstock/meta.yaml"
+tags:
+  - my-custom-tag
+  - zarr
+stores:
+  - id: "small"
+    name: "The cool small Proto Dataset"
+    url: "gs://leap-scratch/data-library/feedstocks/proto_feedstock/small.zarr"
+    rechunking:
+      - path: "gs://some-bucket/small.zarr"
+        use_case: "multiscales"
+  - id: "large"
+    name: "The even cooler large Proto Dataset"
+    url: "gs://leap-scratch/data-library/feedstocks/proto_feedstock/large.zarr"
+```
+
```

Comment:
Maybe worth adding a note that the validation is ran via GH actions and if the user should rely on that or use the CLI first and foremost.